content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -98,6 +102,30 @@ $ dvc remote modify mygdfolder gdrive_client_id <client ID>
 $ dvc remote modify mygdfolder gdrive_client_secret <client secret>
 ```
 
+## Configure the GDrive remote with a service account
```

Comment:
```suggestion
## Configure the Google Drive remote with a service account
```
Same change for the previous header.

Reply:
Also, what's the difference in the previous header? Should it be something like **Configure the Google Drive remote with user credentials** to make the difference clear at first glance?

Reply:
DVC should be configured with different params in config to auth user account (client id and secret should be in the config) and to auth service account (service account email address and .p12 local key file path params in the config). Renaming previous section header sounds great!

Reply:
Thanks for the info.